packages/framework/src/Component/Cron/Config/CronModuleConfig.php

```diff
@@ -95,4 +112,38 @@ public function assignToInstance(string $instanceName): void
     {
         $this->instanceName = $instanceName;
     }
+
+    /**
+     * @return string
+     */
+    public function getReadableFrequency(): string
+    {
+        if ($this->timeHours === '*' && $this->timeMinutes === '*') {
+            return t('Every 5 minutes');
```

Comment:
Yes, by default the cron is running every 5 minutes. And yes, it can be confusing if you will change default crontab setting, but this is a powerful feature that will help to understand when that cron is running.

Reply:
What about "on each run" (or something similar) instead of "every 5 minutes"?